include/gatekeeper_ratelimit.h

```diff
@@ -33,8 +33,32 @@ struct token_bucket_ratelimit_state {
 	uint64_t stamp;
 };
 
+struct log_ratelimit_state {
+	uint32_t ratelimit_level;
```

Comment:
`struct log_ratelimit_state` must be cache aligned to avoid contention between cores.